composer.json

```diff
@@ -26,10 +26,10 @@
     "friendsofphp/php-cs-fixer": "^2.0",
     "ondram/ci-detector": "^2.1 || ^3.5",
     "php-coveralls/php-coveralls": "^2.0",
-    "php-mock/php-mock-phpunit": "^1.1",
+    "php-mock/php-mock-phpunit": "^1.1 || ^2.0",
     "php-parallel-lint/php-parallel-lint": "^1.2",
-    "phpunit/phpunit": "^5.7",
-    "sebastian/environment": "^1.3.4 || ^2.0 || ^3.0",
+    "sminnee/phpunit": "^5.7",
```

Comment:
we should stick to `phpunit/phpunit` and add new supported versions if possible.

Reply:
That's not possible because `sminnee/phpunit-mock-objects` is blocking that atm. See https://github.com/sminnee/phpunit-mock-objects/pull/6 and his reply (and his other replies on this PR).

Reply:
Okay, I can finish your work then ;)
Please give me access to your repository

Reply:
Not sure which one you meant so I gave you access to this one and the phpunit-mock-objects fork :)

Reply:
Thank you, perfect :rocket: 
This was what I expected :)